manifests/bases/image-reflector-controller/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
You likely want to automate updates for this in future releases, this can be done by adding a line to:

https://github.com/fluxcd/flux2/blob/main/.github/workflows/update.yml#L42